pull_request_template.md

```diff
@@ -0,0 +1,44 @@
+name: Check LeetCode Screenshot Proof
+
+on:
+  pull_request:
+    types: [opened, edited, reopened, synchronize]
+
+jobs:
+  screenshot:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check LeetCode Screenshot Proof
+      id: screenshot
+      uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+    - run: |
+        screenshots=$(find . -name '*.png' -or -name '*.jpg' -or -name '*.jpeg')
+        if [ -z "$screenshots" ]; then
+          echo "LeetCode screenshot not found."
+          exit 1
+        fi
+
+## Description
```

Comment:
Remove description and rather add a checkbox that says
"I have given the PR an appropriate name"